src/maker/config.rs

```diff
@@ -14,32 +14,20 @@ pub struct MakerConfig {
     pub port: u16,
     /// RPC listening port
     pub rpc_port: u16,
-    /// Time interval between connection checks
-    pub heart_beat_interval_secs: u64,
-    /// Time interval to ping the RPC backend
-    pub rpc_ping_interval_secs: u64,
-    /// Time interval ping directory server
-    pub directory_servers_refresh_interval_secs: u64,
-    /// Time interval to close a connection if no response is received
-    pub idle_connection_timeout: u64,
     /// Absolute coinswap fee
     pub absolute_fee_sats: Amount,
     /// Fee rate per swap amount in ppb.
     pub amount_relative_fee_ppb: Amount,
     /// Fee rate for timelocked contract in ppb
     pub time_relative_fee_ppb: Amount,
-    /// No of confirmation required for funding transaction
-    pub required_confirms: u64,
-    // Minimum timelock difference between contract transaction of two hops
+    /// Minimum timelock difference between contract transaction of two hops
     pub min_contract_reaction_time: u16,
```

Comment:
Move this one to internal constants. We don't have enough security measures in the protocol to expose this safely to the user.

Reply:
Done in https://github.com/citadel-tech/coinswap/commit/4237e14a6f8457e293117a63dd103a0e5c1c0d3f